packages/bpmn-editor/src/diagram/nodes/Nodes.tsx

```diff
@@ -141,17 +141,16 @@ export const StartEventNode = React.memo(
       [bpmnEditorStoreApi, id]
     );
 
-    const [isMorphingPanelExpanded, setMorphingPanelExpanded] = useState(false);
+    const [isIconExpanded, setMorphingPanelExpanded] = useState(false);
```

Comment:
I wouldn't change the state name here, as `isMorphingPanelExpanded` is clearer

Reply:
Fixed, thanks Luiz! Think that must've been a 'replace all' that affected that by mistake, nice catch! :)